Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM openjdk:17-slim
+FROM eclipse-temurin:17-jdk-alpine
 
-CMD ["/usr/local/openjdk-17/bin/java", "-jar", "/opt/ssdc-rm-exception-manager.jar"]
+CMD ["/opt/java/openjdk/bin/java", "-jar", "/opt/ssdc-rm-exception-manager.jar"]
 
-RUN apt-get update && \
-apt-get -yq install curl && \
-apt-get -yq clean && \
-rm -rf /var/lib/apt/lists/*
```

Comment:
Losing curl on this image breaks the docker-dev health check, we need to keep it or do something else for the local health check.